src/components/ScholarshipAmountField.jsx

```diff
@@ -90,15 +101,29 @@ function ScholarshipAmountField(props) {
           );
         })}
       </RadioGroup>
+      <FormHelperText error>
+        {typeError || minAmountError || maxAmountError ? helperText : ' '}
```

Comment:
Using just `error` and `amountType`:
```suggestion
        {error && !amountType && 'Please choose an option above'}
```